### ISSUE TITLE
panic with return entered on startup

### ISSUE THREAD

**Issue by jcw** [jeelabs/folie#29]
Folie can get into a panic if the return key is pressed immediately after launch, before folie gets to asking about the port to select:

```
$ folie -r
Select the serial port:
  1: /dev/ttyS0
  2: /dev/ttyS1
  3: /dev/ttyS2
  4: /dev/ttyS3
  5: /dev/ttyUSB0
  6: /dev/ttyUSB1
  7: /dev/ttyUSB2
  8: /dev/ttyUSB3
  9: /dev/ttyUSB4
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x49be12]

goroutine 8 [running]:
panic(0x644e20, 0xc420010100)
    /usr/lib/go/src/runtime/panic.go:500 +0x1a1
github.com/chzyer/readline.(*Instance).SetPrompt(0x0, 0x68a6d6, 0x2)
    /home/tve/src/github.com/chzyer/readline/readline.go:170 +0x22
main.wrappedOpen(0xc4200e2000, 0x1, 0x2)
    /home/tve/src/github.com/jeelabs/folie/wrapped.go:78 +0x41a
```

**Comment by jcw** [comment]
This has probably been solved - caused by starting goroutines at the same time, even though one depended on the other initing some global state.

**Comment by jcw** [comment]
Fixed in v2.5.
